mcp_server/server.py

```
@@ -0,0 +1,1794 @@
+            self._stdin_writer.close()
+            await self._stdin_writer.wait_closed()
+        
+        self._process = None
+        self._stdin_writer = None
+        self._stdout_reader = None
+        self._initialized = False
+        logger.debug(f"[Stdio MCP Client] Connection closed")
+
+
+class McpClient:
+    """MCP 客户端，用于连接到其他 MCP 服务器（HTTP 传输）"""
```

Ruff (0.14.8)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)